iceoryx2-ffi/cxx/include/iox2/unique_port_id.hpp

```diff
@@ -13,9 +13,16 @@
 #ifndef IOX2_UNIQUE_PORT_ID_HPP
 #define IOX2_UNIQUE_PORT_ID_HPP
 
+#include "iox/optional.hpp"
 #include "iox2/internal/iceoryx2.hpp"
 
+#include <array>
+
 namespace iox2 {
+
+constexpr uint64_t UNIQUE_PORT_ID_LENGTH = 128;
+using RawIdType = std::array<uint8_t, UNIQUE_PORT_ID_LENGTH>;
```

Comment:
Could we use a simple C-Array here, like `using RawIdType = [uint8_t, UNIQUE_PORT_ID_LENGTH>;` If this code shall ever be certified we will have a huge problem with the `std::array`.

Reply:
Changed to `iox::vector`, the downside is that it needs to be explicitly zeroed with the side, but since this will be only done once per id, or never, the tradeoff is not so large.